pkg/broker/deprovision_subscriber.go

```diff
@@ -66,8 +71,11 @@ func setFailedDeprovisionJob(dao SubscriberDAO, dmsg JobMsg) {
 }
 
 func cleanupDeprovision(instanceID string, dao SubscriberDAO) error {
-	if err := dao.DeleteExtractedCredentials(instanceID); err != nil {
-		log.Error("failed to delete extracted credentials - %v", err)
+	err := dao.DeleteExtractedCredentials(instanceID)
+
+	if err != nil && strings.Contains(err.Error(), etcdKeyNotFoundContents) {
+		log.Debugf("Attempted to delete extracted credentials, but key was not found: [%v]", instanceID)
```

Comment:
Will we ever want to know this in production? That is would we want to see it at INFO level? If not, leave it at debug. Just wondering if this will be useful information or not.

Reply:
It's probably useful in production, I'll bump it up to INFO.